neusomatic/python/call.py

```diff
@@ -79,31 +79,32 @@ def call_variants(net, vartype_classes, call_loader, out_dir, model_tag, use_cud
                     imsave(file_name, non_transformed_matrices[i, :, :, 0:3])
                 true_path[path] = file_name
                 final_preds[path] = [vartype_classes[predicted[i]], pos_pred[i], len_pred[i],
-                                     map(lambda x: round(x, 4), F.softmax(
-                                         outputs1[i, :], 0).data.cpu().numpy()),
-                                     map(lambda x: round(x, 4), F.softmax(
-                                         outputs3[i, :], 0).data.cpu().numpy()),
-                                     map(lambda x: round(x, 4),
-                                         outputs1.data.cpu()[i].numpy()),
-                                     map(lambda x: round(x, 4),
-                                         outputs3.data.cpu()[i].numpy())]
+                                     list(map(lambda x: round(x, 4), F.softmax(
```

Comment:
@marghoob Because in Python 3: map() and filter() return iterators. A quick fix (which may be not the best one is to use list(map(...)). check here: https://docs.python.org/3.0/whatsnew/3.0.html

Reply:
Got it. Thanks.